src/sagemaker/fw_registry.py

```diff
@@ -80,7 +84,8 @@ def registry(region_name, framework=None):
     """
     try:
         account_id = image_registry_map[region_name][framework]
-        return "{}.dkr.ecr.{}.amazonaws.com".format(account_id, region_name)
+        domain_name = "c2s.ic.gov" if region_name == "us-iso-east-1" else "amazonaws.com"
+        return "{}.dkr.ecr.{}.{}".format(account_id, region_name, domain_name)
```

Comment:
should we maybe refactor this logic somewhere?

Reply:
Updated. Wrapped in one function in utils.py